tests/test_patron.py

```diff
@@ -308,3 +310,134 @@ def test_populate_patron_common_fields_student_null_values_success(
     )
     assert not list(patron_xml_record.emails.children)
     assert len(list(patron_xml_record.find_all("user_identifier"))) == 1
+
+
+def test_populate_staff_fields_all_values_success(
+    staff_patron_template, staff_patron_all_values_dict
+):
+    patron_xml_record = populate_staff_fields(
+        staff_patron_template,
+        staff_patron_all_values_dict,
+    )
+    assert patron_xml_record.last_name.string == "Doe"
+    assert patron_xml_record.first_name.string == "Jane"
+    assert patron_xml_record.user_group.string == "27"
+    assert patron_xml_record.user_group["desc"] == "Staff - Lincoln Labs"
+    assert patron_xml_record.line1.string == "AA-B1-11"
+    assert patron_xml_record.phone_number.string == "555-555-5555"
+    assert patron_xml_record.statistic_category.string == "NK"
+    assert (
+        patron_xml_record.statistic_category["desc"]
+        == "LL-Homeland Protection & Air Traffic Con"
+    )
+
+
+def test_populate_staff_fields_null_values_success(
+    staff_patron_template, staff_patron_all_values_dict
+):
+    for key in [k for k in staff_patron_all_values_dict.keys() if k != "MIT_ID"]:
```

Comment:
I'll use this approach on the common fields null test if you don't have concerns